dotnet_mappings.vim

```diff
@@ -0,0 +1,7 @@
+let g:test#csharp#dotnettest#options = '-f netcoreapp2.0'
```

Comment:
Can we add a comment here with the other possible `-f` options?

Reply:
Added the two others I saw in the Rakefile (netstandard1.3 and netcoreapp1.0).  Are there any others I missed?